Apps/Examples/Examples/All Examples/SymbolClusteringExample.swift

```diff
@@ -0,0 +1,166 @@
+import UIKit
+import MapboxMaps
+
+@objc(SymbolClusteringExample)
+
```

Comment:
I know a bunch of our examples do this, but I'd prefer to have these `@objc` annotations immediately preceding the class definitions so that it's more clear that they're connected.